github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
[Suggested Test Cases: Line 75] It would be good to add some unit tests for checking the error handling of the `GetRaw` function.